docs/DEVELOPER.md

```diff
@@ -0,0 +1,61 @@
+Oengus developer environment
+============================
+
+This instruction provides steps on how to set up a development environment to work on Oengus backend in Ubuntu or other
+Debian-based GNU/Linux distributions. See [DOCKER.md](./DOCKER.md) for Docker instructions.
+
+Prerequisites
+-------------
+
+Install packages which are needed for running Oengus. JDK 16 or later is required. Just install the latest JDK available
+in your distribution. For example, for Java 17:
+
+    $ sudo apt install openjdk-17-jdk postgresql
+
+Source code
+-----------
+
+    $ git clone git@github.com:esamarathon/oengusio.git
+    $ cd oengusio
+
+Postgres
+--------
+
+If you haven't used Postgres before, set up a password for the connection to the database:
+
+    $ sudo su - postgres
+    postgres@hostname:~$ psql
+    postgres=# ALTER USER postgres PASSWORD '<your password here>';
+
+Create database `oengus`:
+
+    $ sudo su - postgres
+    postgres@hostname:~$ createdb oengus
```

Comment:
Same thing here, try to keep this as generic as possible. I would also like to see codeblocks used instead of indentation

Reply:
I've dropped the Linux specific command and tried rewording in cc7054fc76931379ac04fbc161426e3aed4e2c9e.